client/platform/desktop/frontend/store/dataset.ts

```diff
@@ -45,29 +46,41 @@ const datasets = ref({} as Record<string, JsonMetaCache>);
 
 const recents = computed(() => (Object.values(datasets.value)));
 
+async function autoDiscover() {
+  datasets.value = {};
+  /* Nothing came from localStorage, try to populate from autodiscovery */
+  const discovered: JsonMeta[] = await ipcRenderer.invoke('autodiscover-data');
+  discovered.forEach((d) => {
+    Vue.set(datasets.value, d.id, hydrateJsonMetaCacheValue(d));
+  });
+}
```

Comment:
Getting an error here on load, guessing settings need to validate before being able to sync on load.
![image](https://user-images.githubusercontent.com/61746913/128012878-04c044ea-4cbb-4789-ac51-95148353bed9.png)
Going and syncing manually using the button works fine afterwards.

So maybe an `await initializedSettings` here would work?